vowpalwabbit/parse_args.cc

```diff
@@ -88,7 +89,18 @@ std::string find_in_path(const std::vector<std::string>& paths, const std::strin
 #endif
   for (const auto& path : paths)
   {
-    std::string full = VW::ends_with(path, delimiter) ? (path + fname) : (path + delimiter + fname);
+    std::string full;
```

Comment:
nit: slightly more verbose than it needs to be, but the readability probably makes up for that. Alternative would be
```
std::string full = path;
if(!vw::ends_with(path, delimiter)) {
  full += delimiter;
}
full += fname;
```